docs/source/tutorial_worlds.rst

```diff
@@ -27,6 +27,24 @@ This section will cover the details of:
 
 3) `handling large datasets using PyTorch Data Loaders <#multiprocessed-pytorch-dataloader>`_
 
+.. note::
+    When a dataset is very large, or requires a lot of preprocessing before a model
+    can use it, you can use our ``PytorchDataTeacher``, which utilizes multiprocessed
+    dataloading for streaming data from disk (rather than loading it into memory).
+    That system can take your task as input, and make it fast to load, or you can
+    roll your own specific setups if you speed or space issues are really at a premium.
+
+First, let's consider a diagram of the basic flow of DialogPartnerWorld,
+a simple world with two conversing agents.
+
+.. image:: _static/img/world_basic.png
+
+The teacher generates a message, which is shown to the agent.
+The agent generates a reply, which is seen by the teacher.
+
+
+Expanding to batching / hogwild using share()
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 For all tasks one might make,
```

Comment:
this text is meant to be a summary of the 3 sections listed, so i dont understand why you've put them out of order and turned the third point into a note, and added another section